benches/bench.rs

```diff
@@ -67,15 +68,21 @@ pub fn jsonrpsee_types_v2(crit: &mut Criterion) {
 		b.iter(|| {
```

Comment:
Perhaps rename/change this and the bench below to bench building named and unnamed params, and also have a bench to set a "baseline" where we just contruct a RawValue?